model/labels/labels_test.go

```diff
@@ -67,6 +62,16 @@ func BenchmarkString(b *testing.B) {
 	}
 }
 
+func TestSizeOfLabels(t *testing.T) {
```

Comment:
we can add a precheck on `testCaseLabels` and  `expectedSizeOfLabels` lens to be future proof.

Reply:
we can also ensure testCaseLabels and expected in TestLabels_String have the same length

Reply:
done